server/services/userService.ts

```diff
@@ -1,20 +1,20 @@
 import { convertToTitleCase } from '../utils/utils'
-import type HmppsAuthClient from '../data/hmppsAuthClient'
 import { NomisUserRolesApi, type Caseload } from '../data/nomisUserRolesApi'
+import ManageUsersApiClient from '../data/manageUsersApiClient'
 
 export interface UserDetails {
```

Comment:
shouldn't this extend `User` from `'../data/manageUsersApiClient'` cos it's calling getUser and adding 3 new fields to it?

Reply:
Good point, done it here: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/646/commits/381be600ac783231e4ed439eb8b881d357ecb8d0